vtex-apps/product-price
# Create a new 'product-seller-name' block

#### What problem is this solving?

We don't currently have a native block that enables users to show the product's seller. Some developers were asking for it in our Office Hours, and this PR aims to give them that ability.

#### How to test it?

Go to this [Workspace](https://sellername--storecomponents.myvtex.com/vintage-phone/p) and you should be able to see a "Sold by: VTEX" text above the product's reference number.

#### Screenshots or example usage:

<img width="658" alt="Screen Shot 2020-09-03 at 10 51 07" src="https://user-images.githubusercontent.com/27777263/92123798-6107b980-edd3-11ea-96ec-88f186ab3ae7.png">

#### Related to / Depends on

Should be released after https://github.com/vtex-apps/product-price/pull/42.

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/3ohc1f29itSXer9ENq/giphy.gif)
